tests/testthat/test-utils_get_db.R

```diff
@@ -79,10 +84,10 @@ test_that("utils_get_db functions other than dbSelect", {
   })
   
   test_that("get_comm_data works", {
-    cmdata <- get_comm_data(pkg_name, app_db_loc)
+    cmdata <- test_pkg_cum[[pkg_name]]
```

Comment:
This test is no longer testing `get_comm_data()`. I guess this could be a different PR...